cmd/core/main.go

```diff
@@ -117,7 +118,7 @@ func main() {
 	}
 
 	logger := zap.New(func(o *zap.Options) {
-		o.Development = true
+		o.Development = logDebug
```

Comment:
so it was true before, do you want to turn it off?

Reply:
Hi @ryanzhang-oss , this was needed to give an additional debug-log flag to the user to turn on/off the lots of debug logs.

This is in result to https://github.com/crossplane/oam-kubernetes-runtime/issues/284#issue-737555765 https://github.com/crossplane/oam-kubernetes-runtime/pull/323#issuecomment-802575866